docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,92 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
+
+#### library scripts
+Please ensure that all library scripts are executed from the `library` directory and a user must first run `npm install` to install all the Libraries dependencies.  
+- `npm run build-dev`: Build the library in development mode
+- `npm run build-prod`: Build the library in production mode
+
+#### example scripts
+Please ensure that all example scripts are executed from the `examples/typescript` directory. It is always recommended that a user installs the library first as the example requires the library. Contrary to this however, `build-all-dev` and `build-all-prod` do not require the library to be installed and built first and will install and build both the library and example and all dependencies.
+- `npm run build-dev`: Build the library in development mode
+- `npm run build-prod`: Build the library in production mode
+- `npm run serve-dev`: Serve the example locally using the library in development mode
+- `npm run serve-prod`: Serve the example locally using the library in production mode
+- `npm run symlink`: Links the library to the example for consumption
+- `npm run build-all-dev`: Build the library and the example in development mode and link the library to the example for consumption
+- `npm run build-all-prod`: Build the library and the example in production mode and link the library to the example for consumption
+
+### Installing the Scalable Pixel Streaming Frontend library from Github
+Please note the following installation will be done with `dev` npm scripts however, it can also be done with `prod` build scripts.
+
 1) Download the [Scalable Pixel Streaming Frontend source code from GitHub](https://github.com/ScalablePixelStreaming/Frontend)
 
-2) Run the following command in the base directory of the source tree:
+2) To build the library run the following commands in the `library` directory of the source tree:
 ```bash
-# Install the frontend library's dependencies
-npm install .
+# Install the Frontend library's dependencies
+npm install 
+
+# Build the Frontend library
+npm run build-dev
 ```
-This will install both the Pixel Streaming Frontend and Frontend UI along with other dependencies which the Scalable Pixel Streaming Frontend consumes. 
+These commands and scripts will install and build the library.
 
-### Running the Scalable Pixel Streaming Frontend locally
-1) To run the Frontend locally after installing simply run the following command from the base directory of the source tree:
+3) Run the following commands in the `examples/typescript` directory of the source tree:
 ```bash
-# Build the frontend library
-npm run build
+# Install the examples's dependencies
```

Comment:
examples' if there are multiple examples with dependencies, or example's if there is one example with dependencies.

Reply:
Resolved in 4820831c542f48909f428f142606b7c814b6ede6